apps/dashboard/src/components/workflow-editor/steps/template-tabs.tsx

```diff
@@ -64,7 +64,7 @@ export const TemplateTabs = ({
           <span className="sr-only">Close</span>
         </CompactButton>
       </header>
-      <TabsContent value="editor" className="h-full w-full overflow-y-auto">
+      <TabsContent value="editor" className="h-full w-full overflow-hidden">
```

Comment:
This ensures that the editor step is never scrollable, but only its subfields (mostly Maily) are.